src/index.ts

```diff
@@ -0,0 +1,17 @@
+import fetchPonyfill from 'fetch-ponyfill'
+import { Configuration as GeneratedConfiguration, ConfigurationParameters } from '../generated/fetch/src/index'
+
+class Configuration extends GeneratedConfiguration {
+  constructor (options: ConfigurationParameters) {
+    /**
+     * Prevent the need for everyone to have to override the fetch API...
```

Comment:
:thought_balloon:  In practice, in which runtimes do we override? Electron?

Reply:
all runtimes. because the API requires everyone to pass a fetch implementation, but we should provide a default. fetchPonyfill is isomorphic.